{"name": "困人事件回放", "status": "passed", "attachments": [{"name": "log", "source": "128bf188-ba69-40f4-a7a5-acfddaaf7f90-attachment.txt", "type": "text/plain"}], "start": 1690256289236, "stop": 1690256289274, "uuid": "29ad57a4-0fb8-4d5c-adf7-bd411a32aa77", "historyId": "573010e86cab1a3d90647b98551bd886", "testCaseId": "8a4e3003043ec7852cf822ca71790ce0", "fullName": "opsPC.test_emergency#test_device_error_trapped_video_replay_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "18760-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}